ui/apps/platform/src/Containers/ComplianceEnhanced/Schedules/ViewScanConfigDetail.tsx

```diff
@@ -29,8 +29,9 @@ import {
 import { getAxiosErrorMessage } from 'utils/responseErrorUtils';
 
 import useFeatureFlags from 'hooks/useFeatureFlags';
-import { JobContextTab } from 'types/reportJob';
+import { jobContextTabs } from 'types/reportJob';
```

Comment:
What are pro and con of reportsJobs.ts files in **types** and **utils** folders versus in src/Components/ReportJobs folder that is being added in #12782

Reply:
I'm not opposed to that suggestion. I think it might be better to keep it there now that it exists.